tests/multigrid/test_netgen_gmg.py

```diff
@@ -0,0 +1,38 @@
+from firedrake import *
+import pytest
+
+
+def test_netgen_mg_circle():
+    try:
+        from netgen.geom2d import Circle, CSG2d
+    except ImportError:
+        pytest.skip(reason="Netgen unavailable, skipping Netgen test.")
```

Comment:
We have a more standard way of doing this. See https://github.com/firedrakeproject/firedrake/blob/master/tests/conftest.py#L22.